### ISSUE TITLE
About the shapenet dataset

### ISSUE THREAD

**Issue by queenie88** [cuge1995/PointCutMix#9]
How to construct the cutmix data for the segmentation task on shapenet dataset. How to mix the points and seg label for the training data? Is it the same with the classification? How to compute the loss？

**Comment by cuge1995** [comment]
We mix points as the same as classification, for the per point label, we keep the original as ground truth.
